.wuf/generator/templates/desktop/Component/Component.st.css

```diff
@@ -1,18 +1,30 @@
 :import {
     -st-from: "../../common/formatters.st";
-    -st-named: color;
+    -st-named: color, fallback;
 }
 
-/*Defaults*/
+/* Defaults - default params */
 :vars {
-    /*Defaults params goes here*/
+  defaultBgColor: color(color-1);
 }
 
-/*Overrides*/
+/* Overrides - deceleration of overridable parameters by the consumer */
 :vars {
-    /*Deceleration of overridable placeholders*/
+  /*
+    Main background color
+  */
+  BgColor: --overridable;
+}
+
+/* Overrides */
+:vars {
+  bgColor: color(fallback(value(BgColor), value(defaultBgColor)));
 }
 
 .root {
-    /*css goes here*/
+  background-color: value(defaultBgColor);
 }
+
+.styleParams { 
+  background-color: value(bgColor);
+}
```

Comment:
even though the TPA's use the `styleParams` to refer to the variables coming from the settings
I'm not sure that in this context it's completely understood

I mean, if this wasn't `css` I think I would go with `applyStyles` or something like that
but then `.applyStyles{ ... }` is a bit weird

idk, mainly thinking out loud

Reply:
Other options can be paramsOverrides or overrideParams.

I want a name that is short yet explicit enough to explain the intent.